gems/quilt_rails/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+## Development
+
+To get started with gem development:
+
+```
+dev clone quilt
+dev cd quilt
+cd gems/quilt_rails
+dev up
+```
+
+For Node development, see [the Node contributors docs](.github/CONTRIBUTING.md).
+
+## Changes
+
+All notable changes should be included in the [`CHANGELOG.md`](CHANGELOG.md).
+
+## Releasing
+
+**Note:** Gem releases will _not_ publish Node packages. If the gem depends on changes in the Node library, you _must_ [publish a Node release first](.github/contributing.md#releasing).
+
+1. `cd gems/quilt_rails`
+1. Update `version.rb` and `CHANGELOG.md` to your new desired version
+1. Commit the resulting changes, `git commit -m "Release v<new-version>"`
```

Comment:
Using `bundle install --without development:test` in CI should make releaseit usable in development.